semantics/src/main/kotlin/com/strumenta/kolasu/semantics/symbol/description/SymbolDescription.kt

```diff
@@ -0,0 +1,39 @@
+package com.strumenta.kolasu.semantics.symbol.description
+
+data class SymbolDescription(
+    val identifier: String,
+    val types: List<String>,
+    val properties: Map<String, ValueDescription>
+)
+
+sealed interface ValueDescription {
```

Comment:
We may also want to use enums or arbitrary primitive types

Reply:
`StringValueDescription`, `IntegerValueDescription` and `BooleanValueDescription` currently support the specification of primitive typed values. These are just wrappers to avoid using a map of `Any` typed properties and maintain control over which kinds of values we handle.

Enumerations can be added fairly easy - I will keep this comment as remainder before merging the PR.